### ISSUE TITLE
Autocomplete

### ISSUE THREAD

**Issue by jorritsmit** [gepd/Deviot#211]
Is there a plan to include autocomplete in the future?

**Comment by gepd** [comment]
Yes, I just did not have time to read that part of the ST documentation. I'll check it too see how much that can took to implement and if it can be done to the next release. I'll let you know the news

**Comment by jorritsmit** [comment]
That would be excellent. Let me know if you want any help!
